pages/linux/nordnm.md

```diff
@@ -0,0 +1,35 @@
+# nordnm
+
+> This tool automates the importing and secures the usage of NordVPN OpenVPN configurations through Network Manager.
+
+- View available categories and countries:
+
+`sudo nordnm list --categories --countries`
+
+- Synchronise, update configuration files, activate the kill-switch and auto-connect to a "normal" UDP server in the US:
```

Comment:
What is the "kill-switch" ? This description is a bit too long. Any chance we can reduce the no. of options ?